CHANGELOG.md

```diff
@@ -1,5 +1,97 @@
 # Changelog
 
+## 0.8.3 
+
+### Breaking Change to JSON RPC-API 
+
+From v0.8.3, incoming HTTP requests are only accepted from hostnames specified using the `--host-whitelist` command-line option. If not specified, the default value for `--host-whitelist` is `localhost`. 
+
+If using the URL `http://127.0.0.1` to make JSON-RPC calls, use `--host-whitelist` to specify the hostname `127.0.0.1` or update the hostname to `localhost`. 
+
+If your application publishes RPC ports, specify the hostnames when starting Pantheon. For example:  
+```json
+pantheon --host-whitelist=example.com
+```
+ 
+Specify `*` or `all` for `--host-whitelist` to effectively disable host protection and replicate pre-v0.8.3 behavior. This is not recommended for production code. 
+
+### Bug Fixes 
+
+- Repair Clique Proposer Selection [\#339](https://github.com/PegaSysEng/pantheon/pull/339) 
+- High TX volume swamps block processing [\#337](https://github.com/PegaSysEng/pantheon/pull/337) 
+- Check if the connectFuture has completed successfully [\#293](https://github.com/PegaSysEng/pantheon/pull/293) 
+- Switch back to Xerial Snappy Library [\#284](https://github.com/PegaSysEng/pantheon/pull/284) 
+- ShortHex of 0 should be '0x0', not '0x' [\#272](https://github.com/PegaSysEng/pantheon/pull/272) 
+- Fix pantheon CLI default values infinite loop [\#266](https://github.com/PegaSysEng/pantheon/pull/266) 
+
+### Additions and Improvements 
+
+- Added `--nodes-whitelist` parameter to CLI and NodeWhitelistController [\#346](https://github.com/PegaSysEng/pantheon/pull/346) 
+- Discovery wiring for `--node-whitelist` [\#365](https://github.com/PegaSysEng/pantheon/pull/365) 
+- Plumb in three more metrics [\#344](https://github.com/PegaSysEng/pantheon/pull/344) 
+- `ProposerSelection` to support multiple IBFT implementations [\#307](https://github.com/PegaSysEng/pantheon/pull/307) 
+- Configuration to support IBFT original and revised [\#306](https://github.com/PegaSysEng/pantheon/pull/306) 
+- Added host whitelist for JSON-RPC. [**Breaking Change**](#breaking-change-to-json-rpc-api) [\#295](https://github.com/PegaSysEng/pantheon/pull/295) 
+- Reduce `Block creation processed cancelled` log message to debug [\#294](https://github.com/PegaSysEng/pantheon/pull/294) 
+- Implement iterative peer search [\#268](https://github.com/PegaSysEng/pantheon/pull/268) 
+- Added RLP enc/dec for PrePrepare, Commit and NewRound messages [\#200](https://github.com/PegaSysEng/pantheon/pull/200) 
+- IBFT block mining [\#169](https://github.com/PegaSysEng/pantheon/pull/169) 
+- Added `--goerli` CLI option [\#370](https://github.com/PegaSysEng/pantheon/pull/370) (Thanks to [Nashatyrev](https://github.com/Nashatyrev))
```

Comment:
To reference a github user put `@` before their name

Reply:
Done